### ISSUE TITLE
[1.11.2 - 1.5.0] Chest render bug & delete of all/most items

### ISSUE THREAD

**Issue by VallenFrostweaver** [CyclopsMC/ColossalChests#78]
I run a server running the Modded Mischeif II pack and have obtained as much information I can think of from the 2 players that have experienced this issue (both are admins as well). No chunk loaders are around as a side note.  If you need more information, please let me know and I will try to get more. As it is, we are currently thinking about removing the mod due to this instability though I would MUCH prefer to keep it. I hope the below helps.

Experience #1: Made a 7x7 wooden chest because 5x5 was full. Had transfered about 90% of it via Xnet. Upon logging back in the 5x5 wasn't rendering (invisible) and had only a single chest of items remaining. Breaking/replacing did not change anything.

Experience #2: 5x5 wooden chest wasn't rendering (invisible). Items disappeared when tried to access chest. Nothing was touching the chest. The chest wouldn't become visible again unless a block had been broken and replaced.

**Comment by rubensworks** [comment]
That's strange, never heard of an issue like this before.

So is this always happening to chests when relogging? Or only sometimes?

**Comment by VallenFrostweaver** [comment]
Sometimes. I can't seem to reliably reproduce. #2 experienced the invisible chest several times before the items went.

**Comment by rubensworks** [comment]
It's possible though that the chest core tile entity somehow gets corrupted, which could be caused by something else outside of this mod.
But I'll see if I can reproduce it.

**Comment by rubensworks** [comment]
@VallenFrostweaver I can't reproduce this issue myself unfortunately.

My only theory is that the core tile entity gets corrupted. This could either be caused by this mod, or something external.
The only way for me to know what goes wrong is to have access to your full forge log, it's possible that there are stacktraces or warnings in there related to tile entity loading or saving.

**Comment by jazzadellic** [comment]
Just came here to actually report this nearly exact bug. My colossal chest became invisible and it started causing all kinds of weird problems. For one, everything on my toolbar and inventory kept disappearing or changing, as did the contents of the chest. I kept reloading my game world to see if it would fix it, and my toolbar would come back, but the chests remained bugged even on world reloading 2x. I finally broke the chest and it appeared to drop mostly what I remembered being in there to begin with. Damn I wish I had taken a screen shot of it. Not only did the colossal chest go invis, but nearby chests from the Quark mod were glitching as well by losing their textures and turning purple and pink squares. I don't know if the two bugs are related or what but the colossal chest was very close, within a few blocks of the Quark chests. What's really weird was the bugging Quark chests when I looked at them said they were spruce chests, but I clearly remembered them being dark oak chests. After I broke them, and set them back down again, indeed they were dark oak chests.
